README.md

```diff
@@ -17,9 +17,9 @@ async function main(stream) {
   const read = consume(stream)
   while (true) {
     const chunk = await read(16)
-    if (!chunk) break
+    if (chunk.done) break
```

Comment:
I'm guessing `chunk` will always be defined and not null, but I didn't dig deep.